app/controllers/proposals_controller.rb

```diff
@@ -1,10 +1,11 @@
 class ProposalsController < ApplicationController
+  skip_authorization_check
```

Comment:
Why?

Reply:
@Ana06 I think we should remove this, isn't  line 8 enough?